### PR TITLE
Restore important pony functionality

### PR DESCRIPTION
**Pull request by DavidStirling**
Someone should probably test this on Mac before merging.

### CODE REVIEW

File: cellprofiler/gui/preferencesdlg.py [hunk 1]
```diff
@@ -456,7 +456,7 @@ def get_preferences(self):
                 cellprofiler_core.preferences.get_wants_pony,
                 cellprofiler_core.preferences.set_wants_pony,
                 CHOICE,
-                "Pony",
+                "The end is neigh.",
```
Comment (AetherUnbound) [inline cellprofiler/gui/preferencesdlg.py:459]: 😂 💯

**Comment by 0x00b1** [comment]
Merged. @DavidStirling Can you post a link to the license so we have it recorded?

**Comment by DavidStirling** [comment]
It's modified from [here](https://freesound.org/people/foxen10/sounds/149024/). License is CC0.

**Comment by 0x00b1** [comment]
Nice. Thanks, @DavidStirling. :)

**Pull request merged**